<launch>
  <!-- ImageProcessor server -->
  <arg name="path"/>
  <node name="ImageProcessor" pkg="drawbot" type="ImageProcessor.py" args="$(find drawbot)/$(arg path)" />

  <!-- Visualization -->
 <node name="DotVisualizer" pkg="drawbot" type="DotVisualizer.py" output="screen"/>

</launch>
